### DISCUSSION TITLE
Dex Share Follower BG delays

### DISCUSSION THREAD

**Discussion by Bwangster12** [Navid200/xDrip#78] [General]
I just installed xDrip+ on a tablet and my phone in order to follow my daughter's Dexcom G6 app. I chose the Dex Share Follow option and am finding that very often they readings are delayed to the xDrip apps... On both devices. I see the status screen shows polling times, and often the BG delay shows 5 or more minutes... Whereas another device Dexcom Follow app or her G6 app already has been updated.

**Comment by Navid200** [comment]
Thanks for posting here.
Would you please have a look at this issue?
https://github.com/NightscoutFoundation/xDrip/issues/1849

Is it related to your concern?
